test/testwindow.go

```diff
@@ -139,24 +124,3 @@ func (w *testWindow) SetContent(obj fyne.CanvasObject) {
 func (w *testWindow) Canvas() fyne.Canvas {
 	return w.canvas
 }
-
-// NewWindow creates and registers a new window for test purposes
-func NewWindow(content fyne.CanvasObject) fyne.Window {
```

Comment:
Rather than removing this API can you wrap it to the new setup? (then some test changes in this refactor can be reverted)

Reply:
Where is the advantage in doing so?
I think it's better to use the normal Fyne API to setup test environment instead of a special test API.
Also this would only save like five lines while hiding the dependency from the (current) app.

Reply:
The advantage is that we don’t remove a public API that is in previous releases